com.dhsdevelopments.Climaxima.json

```diff
@@ -0,0 +1,170 @@
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://git.code.sf.net/p/maxima/code",
+                    "commit": "0721ffcd43de13fd288362373edc9681929abed9"
+                }
+            ],
+            "build-options": {
+                "env": {
+                    "SBCL_HOME": "/app/sbcl/usr/local/lib/sbcl"
+                }
+            },
+            "config-opts": [ "--enable-sbcl", "--with-sbcl=/app/sbcl/usr/local/bin/sbcl" ],
+            "build-commands": [ "cp -ar . /app/maxima-code",
+                                "ln -s /app/maxima-code /app/quicklisp/local-projects/maxima-code" ]
+        },
+        {
+            "name": "climaxima",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/lokedhs/maxima-client",
+                    "commit": "3c198d28f5b036a7b584387a2e2480869026a664"
+                },
+                { "type": "file", "path": "startup.lisp" },
+                { "type": "file", "path": "build.sh" }
+            ],
+            "build-options": {
+                "env": {
+                    "SBCL_HOME": "/app/sbcl/usr/local/lib/sbcl"
+                },
+                "build-args": [
+                    "--share=network"
+                ]
+            },
+            "build-commands": [
+                "cp -ar . /app/maxima-client-build",
+                "ln -s /app/maxima-client-build /app/quicklisp/local-projects/maxima-client-build",
+                "cd /app/maxima-client-build && sh build.sh",
+                "mkdir /app/maxima-client",
+                "cp /app/maxima-client-build/clim-maxima /app/maxima-client",
+                "cp -ar /app/maxima-client-build/fonts /app/maxima-client-build/images /app/maxima-client",
+                "mkdir /app/maxima-client/infoparser",
+                "cp -ar /app/maxima-client-build/infoparser/docs /app/maxima-client-build/infoparser/figures /app/maxima-client/infoparser",
+                "mkdir -p /app/bin",
+                "ln -s /app/maxima-client/clim-maxima /app/bin/clim-maxima"
+            ]
+        },
+        {
+            "name": "desktop-integration",
+            "buildsystem": "simple",
+            "sources": [ { "type": "file", "path": "com.dhsdevelopments.Climaxima.appdata.xml" },
+                         { "type": "file", "path": "com.dhsdevelopments.Climaxima.desktop" } ],
+            "build-commands": [
+                "mkdir -p /app/share/metainfo",
+                "cp com.dhsdevelopments.Climaxima.appdata.xml /app/share/metainfo/com.dhsdevelopments.Climaxima.appdata.xml",
+                "mkdir -p /app/share/applications",
+                "cp com.dhsdevelopments.Climaxima.desktop /app/share/applications/com.dhsdevelopments.Climaxima.desktop"
+            ]
+        }
+    ],
+    "finish-args": [
+        "--socket=x11",
```

Comment:
if you use --socket=x11 then you should add --share=ipc as well http://docs.flatpak.org/en/latest/sandbox-permissions.html

Reply:
Thank you. I have made this change.